src/components/body/body-cell.component.ts

```diff
@@ -334,4 +401,10 @@ export class DataTableBodyCellComponent implements DoCheck, OnDestroy {
     return html.replace(/<\/?[^>]+(>|$)/g, '');
   }
 
+  onTreeAction(row: any) {
```

Comment:
If you move that element to a button and add a disabled property this will be handled for you automatically.